push-build.sh

```diff
@@ -1348,6 +1348,16 @@ release::gcs::publish () {
   fi
 }
 
+ensure_crane_exists () {
+  if ! command -v crane &> /dev/null; then
+    echo "Installing crane..."
+    CRANE_VERSION="v0.19.0"
```

Comment:
can you add this to the dependencies.yaml to we track the version?

Reply:
Done